src/XamlStyler/MarkupExtensions/Formatter/AttributeInfoFactory.cs

```diff
@@ -10,21 +11,47 @@ public class AttributeInfoFactory
     {
         private readonly AttributeOrderRules orderRules;
         private readonly MarkupExtensionParser parser;
+        private readonly IList<string> ignoredNamespacesLocalNames;
+        private readonly bool ignoreDesignTimeReferencePrefix;
 
-        public AttributeInfoFactory(MarkupExtensionParser parser, AttributeOrderRules orderRules)
+        public AttributeInfoFactory(MarkupExtensionParser parser, AttributeOrderRules orderRules, IList<string> ignoredNamespacesLocalNames, bool ignoreDesignTimeReferencePrefix)
         {
             this.parser = parser;
             this.orderRules = orderRules;
+            this.ignoredNamespacesLocalNames = ignoredNamespacesLocalNames;
+            this.ignoreDesignTimeReferencePrefix = ignoreDesignTimeReferencePrefix;
         }
 
         public AttributeInfo Create(XmlReader xmlReader)
         {
             string attributeName = xmlReader.Name;
             string attributeValue = xmlReader.Value;
-            AttributeOrderRule orderRule = this.orderRules.GetRuleFor(attributeName);
+            
+            var attributeNameWithoutNamespace = string.Empty;
```

Comment:
nit. **S**tring.Empty

Reply:
Could you tell me why String would be better fit? IDE suggests me to replace String with string